template.yaml

```diff
@@ -306,6 +306,20 @@ Resources:
                   - dynamodb:PutItem
                   - dynamodb:Query
                 Resource: !GetAtt DynamoDBTable.Arn
+        - PolicyName: s3bucket
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:Put*
+                  - s3:Get*
+                  - s3:List*
+                  - s3:Delete*
+                Resource:
+                  - !Join
+                    - ''
+                    - - 'arn:aws:s3:::'
```

Comment:
We should scope this to the AWS Account Id

Reply:
It's already scoped to the s3 bucket name. Keeping this.